docs/quickstart/install-and-use-a-package-in-visual-studio.md

```diff
@@ -1,78 +1,86 @@
 
-Visual Studio creates the project, which can be found in Solution Explorer. For more information, see [Learn about Solution Explorer](/visualstudio/ide/use-solution-explorer).
+   Visual Studio creates the project, and it appears in [Solution Explorer](/visualstudio/ide/use-solution-explorer).
 
 ## Add the Newtonsoft.Json NuGet package
 
-To install the package, you can use either the NuGet Package Manager or the Package Manager Console. When you install a package, NuGet records the dependency in either your project file or a `packages.config` file (depending on the project format). For more information, see [Package consumption overview and workflow](../consume-packages/Overview-and-Workflow.md).
+To install a NuGet package, you can use either the NuGet Package Manager or the Package Manager Console. Depending on your project format, the installation of a NuGet package records the dependency in either your project file or a *packages.config* file. For more information, see [Package consumption workflow](../consume-packages/overview-and-workflow.md).
 
 ### NuGet Package Manager
 
-1. In Solution Explorer, right-click **References** and choose **Manage NuGet Packages**.
+To use the [NuGet Package Manager](../consume-packages/install-use-packages-visual-studio.md) to install the `Newtonsoft.Json` package in Visual Studio, follow these steps:
 
-    ![Manage NuGet Packages command for project References](media/QS_Use-02-ManageNuGetPackages.png)
+1. In **Solution Explorer**, right-click **References** and choose **Manage NuGet Packages**.
 
-1. Choose "nuget.org" as the **Package source**, select the **Browse** tab, search for **Newtonsoft.Json**, select that package in the list, and select **Install**:
+    :::image type="content" source="media/qs-use-01-manage-nuget-packages.png" alt-text="Screenshot showing Solution Explorer with References selected and Manage NuGet Packages highlighted in shortcut menu.":::
 
-    ![Locating Newtonsoft.Json package](media/QS_Use-03-NewtonsoftJson.png)
+1. Choose **nuget.org** as the **Package source**.
 
-    If you want more information on the NuGet Package Manager, see [Install and manage packages using Visual Studio](../consume-packages/install-use-packages-visual-studio.md).
+1. From the **Browse** tab, search for *Newtonsoft.Json*, select **Newtonsoft.Json** in the list, and then select **Install**.
 
-1. Accept any license prompts.
+    :::image type="content" source="media/qs-use-02-install-package.png" alt-text="Screenshot showing the NuGet Package Manager window with the Newtonsoft.Json package selected.":::
 
-1. (Visual Studio 2017 only) If prompted to select a package management format, select **PackageReference in project file**:
+1. (Visual Studio 2017 only) If you're prompted to select a package management format, select **PackageReference in project file**:
```

Comment:
This behavior is true starting with 2017.  We should probably update this to reflect that.